base/shared/src/main/scala/scalaz/typeclass/GroupInstances.scala

```diff
@@ -0,0 +1,12 @@
+package scalaz
+package typeclass
+
+trait GroupInstances {
+
+  implicit val int: Group[Int] = new GroupClass[Int] {
```

Comment:
This implies that `Int`'s (semi)group/monoid instance is (0, +), not (1, *). 

Is that intended?

Reply:
good question, I have no answer for it, except that perhaps it is not time yet to add instances

Reply:
We sure could use some tagging that @S11001001 worked on in Scalaz 7.